### PR TITLE
Ensure GAMS can run all the benchmarks

### PR DESCRIPTION
**Pull request by olejandro**

### CODE REVIEW

**Comment by olejandro** [comment]
#181 would need to be finished first

**Comment by olejandro** [comment]
Also there are `Element is redifined` errors which are caused by multiple descriptions for the same processes / commodities.

**Comment by Antti-L** [comment]
What is the reasoning for including in `times_prc_sets` only "DMD", "CHP", "ELE", "HPL", "IRE", "NST", "PRE", "STG", "STK", "STS"?  Why not include other types as well, e.g. DISTR, MISC, PRV, PRW, REF, XTRACT, RENEW?  Some of these even have special meaning in the TIMES code, and therefore models would not work correctly if you exclude them.

**Comment by olejandro** [comment]
Thanks, @Antti-L. No particular reason, except for the other types not present in any of the benchmark and the ease with which the list can be expanded. I can include all of the already. I.e. all members of `SET PRC_GRP` in `maplists.def`?

File: xl2times/datatypes.py [hunk 1]
```diff
@@ -205,6 +206,7 @@ def __init__(
             self.all_attributes,
             param_mappings,
         ) = Config._process_times_info(times_info_file)
+        self.times_sets = Config._read_times_sets(times_sets_file)
```
Comment (siddharth-krishna) [inline xl2times/datatypes.py:209]: We should also add a type annotation for `times_sets` after line 193

**Review by siddharth-krishna — APPROVED** [review]

**Pull request merged**